Source/wave.cpp

```diff
@@ -25,7 +25,7 @@ void __fastcall WGetFileArchive(HANDLE hsFile, unsigned int *retries, const char
 	if (*retries >= 5)
 		FileErrDlg(FileName);
 
-	if (hsFile && SFileGetFileArchive(hsFile, &archive) && archive != diabdat_mpq) {

```

Comment:
This breaks bin exactness, maybe the header for SFileGetFileArchive is wrong instead?